README.md

```diff
@@ -61,7 +61,10 @@ Add the following to a file called `bazel_versions.bzl` at the root of your repo
 Bazel version values with the values that you would like to test against for your integration tests.
 
 ```python
-CURRENT_BAZEL_VERSION = "5.1.0"
+# If you are using Bazelisk for your project, you can specify that the current 
+# Bazel version be retrieved from the `.bazelversion` file. Otherwise, specify
+# a valid semantic version.
+CURRENT_BAZEL_VERSION = "//:.bazelversion"
```

Comment:
Great point. I will definitely add a warning. 

I am curious. What are everyone's thoughts on what should be supported from `.bazelversion`? 
- Special Names: Specifying `latest`, `latest_green` or any of the other special names could exhibit a false positive if test runs are cached. 
- Floating versions: Specifying `4.x` has the same problem as above.
- Forks: I think supporting the `<FORK>/<VERSION>` syntax could be useful. 
- Git Hash Commits: This could be useful.

CC: @thesayyn